package.json

```diff
@@ -19,8 +19,8 @@
       "ccdUrl": "http://ccd-data-store-api-demo.service.core-compute-demo.internal"
     },
     "aat": {
-      "cosUrl": "http://prl-cos-aat.service.core-compute-aat.internal",
-      "ccdUrl": "http://ccd-data-store-api-aat.service.core-compute-aat.internal"
+      "cosUrl": "http://prl-cos-pr-840-java",
+      "ccdUrl": "http://prl-ccd-definitions-pr-830-ccd-data-store-api"
```

Comment:
Update here. And also make change in cos PR pointing to CCD. take example of 784 an 584

Reply:
Do not forget to revert this change before merging to master